org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.releng/pom.xml

```diff
@@ -5,9 +5,18 @@
 	<version>2.17.0-SNAPSHOT</version>
 	<artifactId>org.eclipse.xtext.example.domainmodel.parent</artifactId>
 	<packaging>pom</packaging>
-
+	<dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.eclipse.xtext</groupId>
+                <artifactId>xtext-dev-bom</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
```

Comment:
Insonsistent indentation. Tabs vs spaces.

Reply:
Thanks for the feedback @szarnekow ! I replaced spaces with tabs to use consistent indentation.